### ISSUE TITLE
FFI build process broken

### ISSUE THREAD

**Issue by vladimir-vg** [ktap/ktap#86]
Checked out master, tried to use FFI. Discovered that last commit when ktap successfully compiles with `FFI=1` option and executes samples is this: e7a38ef06bec9a651c9e8bdb3ad66a104210d475

Child commit compiles, but cannot run samples. All newer commits do not pass `make FFI=1` successfully.

**Comment by houqp** [comment]
Yeah, FFI is broken in master right now. Last time I heard, @ktap is working on a complete rewrite of the runtime, so he said it's better to wait for the new version and fix FFI after that.

**Comment by vladimir-vg** [comment]
I just thought that you're unaware of it. Okay, great!
